spec/rubocop/cop/lint/next_without_accumulator_spec.rb

```diff
@@ -34,23 +46,28 @@ def code_with_accumulator(method_name)
 
       it 'accepts next with a value' do
         inspect_source(cop, code_with_accumulator(reduce_alias))
-        expect(cop.offenses).to be_empty
+        expect(cop.offenses.size).to eq(0)
+      end
+
+      it 'accepts next within a nested block' do
+        inspect_source(cop, code_with_nested_block(reduce_alias))
+        expect(cop.offenses.size).to eq(0)
       end
     end
   end
 
-  include_examples 'reduce/inject', :reduce
-  include_examples 'reduce/inject', :inject
+  it_behaves_like 'reduce/inject', :reduce
+  it_behaves_like 'reduce/inject', :inject
 
   context 'given an unrelated block' do
     it 'accepts a bare next' do
       inspect_source(cop, code_without_accumulator(:foo))
-      expect(cop.offenses).to be_empty
+      expect(cop.offenses.size).to eq(0)
```

Comment:
Whoops. I changed this because I'd seen so many instances of `eq(0)`. 😀 Will fix.